src/Symfony/Bridge/PhpUnit/Tests/expectdeprecationfail.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+Test ExpectDeprecationTrait failing tests
+--FILE--
+<?php
+$root = realpath($_SERVER['SYMFONY_PHPUNIT_DIR'] . '/../');
```

Comment:
I'm not that happy with test. I think it is fragile.

Reply:
I think the latest commit has a better way of locating simple-phpunit. Should be way less fragile.